nix-install.sh

```diff
@@ -499,7 +478,11 @@ downloader() {
         get_ciphersuites_for_curl
         _ciphersuites="$RETVAL"
         if [ -n "$_ciphersuites" ]; then
-            _err=$(curl $_retry --proto '=https' --tlsv1.2 --ciphers "$_ciphersuites" --silent --show-error --fail --location "$1" --output "$2" 2>&1)
+            if [ -n "$NIX_INSTALL_FORCE_ALLOW_HTTP" ]; then
```

Comment:
Not that important, but I wonder if it would be nice to have some args array set that based on these conditions, instead of `curl`ing in both branches.

Reply:
Part of my intention is to keep changes to the original rustup source fairly small so we can easily update it ^^